leshan-core/src/main/java/org/eclipse/leshan/core/model/DDFFileParser.java

```diff
@@ -169,7 +169,7 @@ private ResourceModel parseResource(Node item) {
                 case "Time":
                     type = Type.TIME;
                     break;
-                }
+                 }
```

Comment:
@jvermillard  what you mean by WTF?
We removed the OBJECTLINK, as you said it has to be in a different PR.